### ISSUE TITLE
model_SRN_Iter_20000.tar

### ISSUE THREAD

**Issue by 6mo6mo6** [FouriYe/DCRGAN-TMM#1]
Sorry to bother you, when training DCR_GAN, do all datasets select the SRN obtained by 20000 iterations in pre-training, i.e. model_SRN_Iter_20000.tar?

**Comment by FouriYe** [comment]
Yes

**Comment by 6mo6mo6** [comment]
Thank you for your reply, there are still some questions:
1. In pretrain_SRN, the number of iterations of SRN and MN on all datasets is 100000. How is the number of iterations determined? Is it determined by the convergence of the loss function or by some other condition?
2. And why choose model_SRN_Iter_20000.tar as the model for training DCR_GAN?

**Comment by 6mo6mo6** [comment]
Thank you for your reply, there are still some questions:
1. In pretrain_SRN, the number of iterations of SRN and MN on all datasets is 100000. How is the number of iterations determined? Is it determined by the convergence of the loss function or by some other condition?
2. And why choose model_SRN_Iter_20000.tar as the model for training DCR_GAN?

**Comment by FouriYe** [comment]
It just an experiential value. SRN could be used as long as the loss is convergent.

**Comment by 6mo6mo6** [comment]
Thank you for your reply.
